src/app/components/create-assignment/create-assignment.component.ts

```diff
@@ -271,10 +273,12 @@ export class CreateAssignmentComponent implements OnInit, OnDestroy {
       this.fc.rubric.updateValueAndValidity();
       this.fc.rubric.disable();
       this.isRubric = false;
+      this.isNoRubric = true;
```

Comment:
I see you added another variable `isNotRubric` which is the flip of `isRubric`....
You can remove both `isRubric` and `isNotRubric` and instead use the value as it is on the form.
In Html`createAssignmentForm.noRubric.value`
In typescript `this.createAssignmentForm.'noRubric.value`